docs/develop/api-reference/dictionary-collections.md

```diff
@@ -2,23 +2,23 @@
 sidebar_position: 1
 sidebar_label: Dictionary
 title: Dictionary API reference
-description: Learn how to interact with the API for dictionary data types in Momento Serverless Cache.
+description: Learn how to interact with the API for Dictionary Collection Data Types in Momento Serverless Cache.
```

Comment:
As these are not named features of the product, they should not be capitalized IMO.